modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -562,14 +574,12 @@ impl LightningClientModule {
 
         let (send_fee, expiration_delta) = routing_info.send_parameters(&invoice);
 
-        if !send_fee.le(&PaymentFee::SEND_FEE_LIMIT_DEFAULT) {
-            return Err(SendPaymentError::PaymentFeeExceedsLimit(send_fee));
+        if !send_fee.le(&PaymentFee::SEND_FEE_LIMIT) {
```

Comment:
As long as we're touching this, can we change this to use `gt` instead of "not less than or equal to"?

Reply:
The statements are are not necessarily identical in a partial ordering.

Reply:
Does partial ordering matter here? If it does we should really have a comment, it is not obvious to me in what situations that would be relevant. In general it is more readable if there are less negations